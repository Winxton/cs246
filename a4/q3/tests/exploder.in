new 11
init
2 3
3 3
4 3
5 3 
6 3
2 7
3 7
4 7
5 7 
6 7
2 5
6 5
-1 -1
print
steps 2
print
step
print
step
print
