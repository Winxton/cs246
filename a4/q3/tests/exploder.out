___________
___________
___X_X_X___
___X___X___
___X___X___
___X___X___
___X_X_X___
___________
___________
___________
___________
___________
___________
___X___X___
__X_____X__
_X__X_X__X_
__X_____X__
___X___X___
___________
___________
___________
___________
___________
___________
___________
__XX___XX__
_XXX___XXX_
__XX___XX__
___________
___________
___________
___________
___________
___________
___________
___________
_X_X___X_X_
_X__X_X__X_
_X_X___X_X_
___________
___________
___________
___________
___________
